src/platforms/native/configuration/backends/index.mdx

```diff
@@ -11,7 +11,17 @@ The Native SDK can use different backends that are responsible for capturing
 crashes. The backend is configured at build-time, using the `SENTRY_BACKEND`
 CMake option, with support for the following options:
 
-- [`crashpad`](crashpad/), which is the default on Windows and macOS,
-- `breakpad`, which is the default on Linux,
-- `inproc`, the default on Android, and
-- `none`, that disables capturing crashes altogether.
+- [`crashpad`](crashpad/): This uses the out-of-process crashpad handler. It is currently
+  only supported on Desktop OSs, and used as the default on Windows and macOS.
+- `breakpad`: This uses the in-process breakpad handler. It is currently
+  only supported on Desktop OSs, and used as the default on Linux.
```

Comment:
I wouldn’t mention "Desktop OSs" at all. crashpad and breakpad work just fine in docker / containers, which are not desktop :-)
Also breakpad "in theory" also works on Android, but we don’t officially support it, primarily because the Java SDK that wraps it is not capable of forwarding minidumps produced by it. And because we don’t have access to all the system symbols on Android.